src/app/tests/suites/certification/Test_TC_MF_1_4.yaml

```diff
@@ -16,398 +16,157 @@ name: 22.1.4. [TC-MF-1.4] Node Behavior using BCM [DUT - Commissionee]
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic"
+    nodeId2:
+        type: NODE_ID
+        defaultValue: 0xCAFE
     endpoint: 0
+    discriminator:
+        type: INT16U
+        defaultValue: 3840
+    payload:
+        type: CHAR_STRING
+        defaultValue: "MT:-24J0AFN00KA0648G00" # This value needs to be generated automatically
 
 tests:
-    - label: "TH_CR1 starts a commissioning process with DUT_CE"
-      verification:
-          "1. Provision the device using 1st python controller on the raspi (use
-          above instructions) "
-      disabled: true
+    - label: "Reboot target device"
+      cluster: "SystemCommands"
+      command: "Reboot"
+      arguments:
+          values:
+              - name: "discriminator"
+                value: discriminator
+
+    - label: "1 TH_CR1 starts a commissioning process with DUT_CE"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "2.a TH_CR1 opens a commissioning window on DUT_CE"
+      cluster: "AdministratorCommissioning"
+      command: "OpenBasicCommissioningWindow"
+      timedInteractionTimeoutMs: 10000
+      arguments:
+          values:
+              - name: "CommissioningTimeout"
+                value: 1
```

Comment:
This is a _really_ short timeout, which will make this test prone to random failures, especially because we do some other work before actually commissioning.  This number should be much higher.

Reply:
Updated